backend/docker/docker-compose-polkadot.yml

```diff
@@ -32,7 +32,7 @@ services:
       POSTGRES_USER: 'polkastats'
       POSTGRES_PASSWORD: 'polkastats'
     healthcheck:
-      test: ['CMD-SHELL', 'pg_isready -U polkastats']
+      test: [ 'CMD-SHELL', 'pg_isready -U polkastats' ]
```

Comment:
Why do we need spaces after `[` and before `]`?

Reply:
auto formatting of my IDE. reverted.